src/rebar_prv_dialyzer.erl

```diff
@@ -193,9 +194,20 @@ do_update_proj_plt(State, Plt, Output) ->
     end.
 
 proj_plt_files(State) ->
-    BasePltApps = base_plt_apps(State),
+    {BasePltApps, BasePltMods} =
+        case is_update_base_plt_active(State) of
+            false ->
+                {ok, BaseFiles} = plt_files(get_base_plt(State)),
+                BaseMods =
+                    lists:map(fun(File) ->
+                                      list_to_atom(filename:basename(File, code:objfile_extension()))
+                              end, BaseFiles),
+                {[], BaseMods};
+            true  ->
+                {base_plt_apps(State), get_config(State, base_plt_mods, [])}
+        end,
+
     PltApps = get_config(State, plt_extra_apps, []) ++ BasePltApps,
-    BasePltMods = get_config(State, base_plt_mods, []),
```

Comment:
Yeah I understood that, currently I moved that code https://github.com/define-null/rebar3/blob/e0223c71b9ae38648b7ee22779a35f3b742db312/src/rebar_prv_dialyzer.erl#L207. So in order to keep this logic consistent I will add it to https://github.com/define-null/rebar3/blob/e0223c71b9ae38648b7ee22779a35f3b742db312/src/rebar_prv_dialyzer.erl#L205 line as well